<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRelief</class>
 <widget class="QMainWindow" name="FormRelief">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relief Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollImageArea">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>598</width>
         <height>481</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::FieldsStayAtSizeHint</enum>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Width, m</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Height, m</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Rows</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Cols</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="EditWidth">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="EditHeight">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="EditRowCount">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>32</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="EditColCount">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>32</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="EditColorCount">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Color Count</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Garbage Filter </string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="EditGarbageFilter">
             <property name="text">
              <string>1.5</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Color to Legend</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QCheckBox" name="chbColorToLegend">
             <property name="text">
              <string>Use Picture</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>X start, m</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Y start, m</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="EditXStart">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="EditYStart">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnApply">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableColors">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>90</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Color</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Z, m</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFile_Open_Image"/>
    <addaction name="actionFile_Open_Relief"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Save_Legend_As"/>
    <addaction name="actionFile_Load_Legend"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Close"/>
   </widget>
   <widget class="QMenu" name="menuRelief">
    <property name="title">
     <string>Relief</string>
    </property>
    <addaction name="actionRelief_Calc_Discrete_Img"/>
    <addaction name="separator"/>
    <addaction name="actionRelief_Calc_Relief_And_Save_As"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRelief"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile_Open_Image">
   <property name="text">
    <string>Open Image...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFile_Close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRelief_Calc_Discrete_Img">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Calc Discrete Img</string>
   </property>
   <property name="toolTip">
    <string>Calc Discrete Img</string>
   </property>
  </action>
  <action name="actionFile_Save_Legend_As">
   <property name="text">
    <string>Save Legend As...</string>
   </property>
  </action>
  <action name="actionFile_Load_Legend">
   <property name="text">
    <string>Load Legend</string>
   </property>
  </action>
  <action name="actionRelief_Calc_Relief_And_Save_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Calc Relief And Save As...</string>
   </property>
   <property name="toolTip">
    <string>Calc Relief And Save As...</string>
   </property>
  </action>
  <action name="actionFile_Open_Relief">
   <property name="text">
    <string>Open Relief</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tableColors</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>FormRelief</receiver>
   <slot>SlotColorDblClicked(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>857</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableColors</sender>
   <signal>cellPressed(int,int)</signal>
   <receiver>FormRelief</receiver>
   <slot>SlotColorCellPressed(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>925</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>892</x>
     <y>644</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SlotColorDblClicked(QTableWidgetItem*)</slot>
  <slot>SlotColorCellPressed(int,int)</slot>
 </slots>
</ui>
